[2023-09-18T13:47:18.617+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [queued]>
[2023-09-18T13:47:18.637+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [queued]>
[2023-09-18T13:47:18.638+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T13:47:18.666+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-14 00:00:00+00:00
[2023-09-18T13:47:18.675+0000] {standard_task_runner.py:57} INFO - Started process 4473 to run task
[2023-09-18T13:47:18.681+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-14T00:00:00+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmpb3wbrnjc']
[2023-09-18T13:47:18.682+0000] {standard_task_runner.py:85} INFO - Job 257: Subtask task2
[2023-09-18T13:47:18.768+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T13:47:18.914+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-14T00:00:00+00:00'
[2023-09-18T13:47:18.916+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T13:47:18.918+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2"]
[2023-09-18T13:47:18.942+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T13:47:18.945+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: unexpected EOF while looking for matching `''
[2023-09-18T13:47:18.946+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 2: syntax error: unexpected end of file
[2023-09-18T13:47:18.947+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-09-18T13:47:18.968+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-09-18T13:47:18.976+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=6.1-monitoring, task_id=task2, execution_date=20230914T000000, start_date=20230918T134718, end_date=20230918T134718
[2023-09-18T13:47:18.997+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 257 for task task2 (Bash command failed. The command returned a non-zero exit code 2.; 4473)
[2023-09-18T13:47:19.014+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-18T13:47:19.094+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-18T13:53:20.030+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [queued]>
[2023-09-18T13:53:20.080+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [queued]>
[2023-09-18T13:53:20.080+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T13:53:20.110+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-14 00:00:00+00:00
[2023-09-18T13:53:20.117+0000] {standard_task_runner.py:57} INFO - Started process 5084 to run task
[2023-09-18T13:53:20.122+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-14T00:00:00+00:00', '--job-id', '403', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmpcr467268']
[2023-09-18T13:53:20.124+0000] {standard_task_runner.py:85} INFO - Job 403: Subtask task2
[2023-09-18T13:53:20.210+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T13:53:20.365+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-14T00:00:00+00:00'
[2023-09-18T13:53:20.369+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T13:53:20.371+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2'"]
[2023-09-18T13:53:20.390+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T13:53:22.397+0000] {subprocess.py:93} INFO - Task 2
[2023-09-18T13:53:22.398+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-18T13:53:22.429+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=6.1-monitoring, task_id=task2, execution_date=20230914T000000, start_date=20230918T135320, end_date=20230918T135322
[2023-09-18T13:53:22.468+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-18T13:53:22.498+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-18T13:55:16.306+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [queued]>
[2023-09-18T13:55:16.395+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [queued]>
[2023-09-18T13:55:16.397+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T13:55:16.469+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-14 00:00:00+00:00
[2023-09-18T13:55:16.491+0000] {standard_task_runner.py:57} INFO - Started process 5283 to run task
[2023-09-18T13:55:16.508+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-14T00:00:00+00:00', '--job-id', '451', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmp16lcutlp']
[2023-09-18T13:55:16.519+0000] {standard_task_runner.py:85} INFO - Job 451: Subtask task2
[2023-09-18T13:55:16.720+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-14T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T13:55:17.024+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-14T00:00:00+00:00'
[2023-09-18T13:55:17.029+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T13:55:17.033+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2'"]
[2023-09-18T13:55:17.058+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T13:55:19.069+0000] {subprocess.py:93} INFO - Task 2
[2023-09-18T13:55:19.071+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-18T13:55:19.136+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=6.1-monitoring, task_id=task2, execution_date=20230914T000000, start_date=20230918T135516, end_date=20230918T135519
[2023-09-18T13:55:19.211+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-18T13:55:19.290+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
